### ISSUE TITLE
Invalid path alias: @sproutbaseemail/console/controllers

### ISSUE THREAD

**Issue by danystad** [barrelstrength/craft-sprout-fields#83]
### Description
After updating Sprout Fields to 3.2.4, I get the following error when running the ./craft command in the console.

`Exception 'yii\base\InvalidArgumentException' with message 'Invalid path alias: @sproutbaseemail/console/controllers'`

### Steps to reproduce

1. Updated Sprout Fields to 3.2.4
2. Trying to run the Craft CLI in the console

### Additional info

- Craft version: 3.1.15
- PHP version: 7.2.14
- Database driver & version: MySQL 5.5.5

**Comment by BenParizek** [comment]
@webmarks There may be more to the story here. I'm unable to recreate the issue simply running `./craft` with Sprout Fields 3.2.4 installed. And it also appears the error message references a Sprout Email path `@sproutbaseemail`.  I'm also unable to recreate the issue with the latest Sprout Email installed. Do you also have Sprout Forms installed (I've tested this too and am not seeing the same error, but it is the only other plugin we have that uses the `sproutbaseemail` module)?

Can you clear caches and try the command again?  Have you had Sprout Email or some other related plugin installed recently?

**Comment by danystad** [comment]
@BenParizek The only other Sprout plugins I had installed is Sprout Encode Email. Now, I just tried on a fresh install of Craft 3.1.15 with only Sprout Fields 3.2.4 installed (no other plugins) and get the same error. There's a Stack trace that comes with the error message in the console. Would that help?

**Comment by andrelopez** [comment]
That would be great @webmarks Could you please paste the stack trace in a new comment?

**Comment by danystad** [comment]
Hi @andrelopez Here is the full stack trace. Hope this helps. Let me know if you need anything else. Thank you!

```
 /Websites/craft3  ./craft
Exception 'yii\base\InvalidArgumentException' with message 'Invalid path alias: @sproutbaseemail/console/controllers'

in /Websites/craft3/vendor/yiisoft/yii2/BaseYii.php:154

Stack trace:
#0 /Websites/craft3/vendor/yiisoft/yii2/base/Module.php(257): yii\BaseYii::getAlias('@sproutbaseemai...')
#1 /Websites/craft3/vendor/yiisoft/yii2/console/controllers/HelpController.php(246): yii\base\Module->getControllerPath()
#2 /Websites/craft3/vendor/yiisoft/yii2/console/controllers/HelpController.php(241): yii\console\controllers\HelpController->getModuleCommands(Object(barrelstrength\sproutbasefields\SproutBaseFields))
#3 /Websites/craft3/vendor/yiisoft/yii2/console/controllers/HelpController.php(175): yii\console\controllers\HelpController->getModuleCommands(Object(craft\console\Application))
#4 /Websites/craft3/vendor/yiisoft/yii2/console/controllers/HelpController.php(187): yii\console\controllers\HelpController->getCommands()
#5 /Websites/craft3/vendor/yiisoft/yii2/console/controllers/HelpController.php(293): yii\console\controllers\HelpController->getCommandDescriptions()
#6 /Websites/craft3/vendor/yiisoft/yii2/console/controllers/HelpController.php(67): yii\console\controllers\HelpController->getDefaultHelp()
#7 [internal function]: yii\console\controllers\HelpController->actionIndex(NULL)
#8 /Websites/craft3/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#9 /Websites/craft3/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#10 /Websites/craft3/vendor/yiisoft/yii2/console/Controller.php(148): yii\base\Controller->runAction('', Array)
#11 /Websites/craft3/vendor/yiisoft/yii2/base/Module.php(528): yii\console\Controller->runAction('', Array)
#12 /Websites/craft3/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('', Array)
#13 /Websites/craft3/vendor/craftcms/cms/src/console/Application.php(93): yii\console\Application->runAction('', Array)
#14 /Websites/craft3/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('', Array)
#15 /Websites/craft3/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
#16 /Websites/craft3/craft(22): yii\base\Application->run()
#17 {main}
```

**Comment by andrelopez** [comment]
Thanks @webmarks 

We were able to recreate this issue and is fixed in Sprout Fields v3.2.5.

Please update your plugin and let us know how it goes.

**Comment by danystad** [comment]
I updated Sprout Fields to v3.2.5 and the Craft CLI is working again. All good!
Thanks for fixing it so quickly @andrelopez
